scylla-rust-wrapper/src/cass_types.rs

```
@@ -532,21 +573,18 @@ pub fn get_column_type(column_type: &ColumnType) -> CassDataType {
                     )
                 })
                 .collect(),
-            keyspace: keyspace.clone().into_owned(),
-            name: type_name.clone().into_owned(),
+            keyspace: definition.keyspace.clone().into_owned(),
+            name: definition.name.clone().into_owned(),
             frozen: false,
         }),
-        ColumnType::SmallInt => CassDataTypeInner::Value(CassValueType::CASS_VALUE_TYPE_SMALL_INT),
-        ColumnType::TinyInt => CassDataTypeInner::Value(CassValueType::CASS_VALUE_TYPE_TINY_INT),
-        ColumnType::Time => CassDataTypeInner::Value(CassValueType::CASS_VALUE_TYPE_TIME),
-        ColumnType::Timeuuid => CassDataTypeInner::Value(CassValueType::CASS_VALUE_TYPE_TIMEUUID),
         ColumnType::Tuple(v) => CassDataTypeInner::Tuple(
             v.iter()
                 .map(|col_type| Arc::new(get_column_type(col_type)))
                 .collect(),
         ),
-        ColumnType::Uuid => CassDataTypeInner::Value(CassValueType::CASS_VALUE_TYPE_UUID),
-        ColumnType::Varint => CassDataTypeInner::Value(CassValueType::CASS_VALUE_TYPE_VARINT),
+
+        // ColumnType is non_exhaustive.
+        _ => panic!("Unknown column type"),
```

:warning: :thinking: Is it reasonable to make cpp-rust-driver panic on an unknown type? Addition of a new type in Rust driver can be skipped upon bumping its version in cpp-rust-driver; then the Rust driver may return a type that is unknown here and OOOOPS...

---

True. I think it's much better to return some error here. The caller can then handle it appropriately.

I'll adjust it in a separate commit(s).

---

Ok, I started doing that and error handling becomes quite messy and complicated (especially when constructing cluster metadata structures). I'm not sure if this should be a part of this PR. I'll open an issue about it.

---

If you prefer not to clutter this PR with that work, let's just return a generic error kind. It's still much better than having this merged with a panic here.